tests/test_gradio.py

```
@@ -0,0 +1,107 @@
+
+    # Create and launch simple interface
+    demo = create_simple_interface()
+
+    print("\n🚀 Launching simple test interface...")
+    print("Open your browser to test CUDA functionality")
+
+    try:
+        demo.launch(share=True, debug=True)
+    except KeyboardInterrupt:
+        print("\n👋 Test stopped by user")
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)